src/pages/signupclear.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
+import HomeLayout from '../components/common/layout/layout';
+
+import { SingupClearContainer, SignupBeerImage, Text, RecommendButton, RecommendText } from '../styles/signupclear';
 
 const Signupclear = () => {
-  return <div>Signupclear</div>;
+  return (
+    <HomeLayout>
+      <SingupClearContainer>
+        <Text>회원가입 완료!</Text>
+        <RecommendButton primary size="small">
+          <RecommendText>맥주추천 받으러 가기</RecommendText>
+        </RecommendButton>
+        <SignupBeerImage src={'/signupBeerIcon.svg'} />
+      </SingupClearContainer>
+    </HomeLayout>
+  );
```

Comment:
레이적용은 `getLayout`함수를 사용해서 적용할 수 있어요!
```suggestion

      <SingupClearContainer>
        <Text>회원가입 완료!</Text>
        <RecommendButton primary size="small">
          <RecommendText>맥주추천 받으러 가기</RecommendText>
        </RecommendButton>
        <SignupBeerImage src={'/signupBeerIcon.svg'} />
      </SingupClearContainer>

  );
```